tests/test_health.py

```diff
@@ -32,4 +32,7 @@ def test_health_resource_query(self):
             'resource': 'health-events',
             'query': [{'services': 'EC2'}]}, session_factory=session_factory)
         resources = p.run()
-        self.assertEqual(len(resources), 5)
+        self.assertEqual(len(resources), 8)
```

Comment:
at some point i think we just want to hand modify the output to trim the resource count here just to avoid extraneous data files in the test dir.